tests/framework/CodeInterfaceTests/RAVEN/tests

```diff
@@ -5,12 +5,28 @@
    UnorderedCsv = 'Basic/outer_out.csv'
    max_time = 500
  [../]
+ [./Code]
+   type = 'RavenFramework'
+   input = 'code.xml'
+   UnorderedCsv = 'Code/outer_out.csv'
+   rel_err = 0.0001
+    [./folderExist]
+      type = Exists
+      output = 'Code/sample/1/innerRunDir/sample/4/out~input.csv '
```

Comment:
Same input for the test, the first test will check the existence of the subdirectory of 5 levels deep.